analysis/import/SourceCodeParser/src/main/kotlin/de/maibornwolff/codecharta/importer/sourcecodeparser/SourceCodeParserMain.kt

```diff
@@ -49,10 +54,11 @@ class SourceCodeParserMain(private val outputStream: PrintStream) : Callable<Voi
     @Throws(IOException::class)
     override fun call(): Void? {
 
+        print(" ")
```

Comment:
Looks like we will do this in all `call()`-Functions. We should do that at one spot.